QA_Requests/BenchmarkTests/OutputFiles/manual_records.csv

```diff
@@ -0,0 +1,44 @@
+property_count, load_time
```

Comment:
I kept the Output files, incase someone wants to verify the results. In case this results in a heavy framework, these files/folder can be deleted.

Reply:
So it'd be fine to delete the complete directory `OutputFiles`? 

@kbo001

Reply:
Yes. I don't think anyone will check the output in CI. If someone wants to check he/she has to execute the scripts on local. Otherwise these output files will eat up a lot of space.
@kiran-tripathy